backend/src/db/migrations/20230914101723_remove_unique_constraint_in_chats_name_column.ts

```diff
@@ -0,0 +1,21 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'chats';
+
+const ColumnName = {
+  NAME: 'name',
+} as const;
+
+function up(knex: Knex): Promise<void> {
+  return knex.schema.table(TABLE_NAME, (table) => {
+    table.dropUnique([ColumnName.NAME]);
+  });
+}
+
+function down(knex: Knex): Promise<void> {
+  return knex.schema.table(TABLE_NAME, (table) => {
+    table.unique([ColumnName.NAME]);
```

Comment:
we need to handle case here when we have non unique items, maybe query non unique items and update their names with index

Reply:
ok, I will handle it